modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,264 @@ Runtime Fabric enables you to specify the level of severity of the message writt
+|===
+| Key | Description | Required | Default value
+| Match | Specifies the pattern to match for tags of logs to be outputted by this plugin. | |
+| Host | Specifies the IP address or hostname of the target Graylog server. | | 127.0.0.1
+| Port | Specifies the port on which your Graylog GELF input is listening. | | 12201
+| Mode | Specifies the protocol to use (tls, tcp or udp). | | udp
+| Gelf_Short_Message_Key | Specifies a short descriptive message (MUST be set in GELF). | | short_message
+| Gelf_Timestamp_Key | Specifies your log timestamp (SHOULD be set in GELF). | | timestamp
+| Gelf_Host_Key | Specifies the key for which the value is used as the name of the host, source, or application that sent the message. (MUST be set in GELF). | | host
+| Gelf_Full_Message_Key | Specifies the key to use as the long message that can contain a backtrace. (Optional in GELF) | |full_message
+| Gelf_Level_Key | Specifies the key to be used as the log level. Its value must be in standard syslog levels (between 0 and 7). | (Optional in GELF) | level
+| Packet_Size | Specifies the size of packets to be sent if the transport protocol is udp. | | 1420
+| Compress | Specifies compression of your UDP packets if the transport protocol is udp. | | true
+|===
+
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name                    gelf
+    Match                   kube.*
+    Host                    <your-graylog-server>
+    Port                    12201
+    Mode                    tcp
+    Gelf_Short_Message_Key  data
+```
+
+==== Splunk Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Splunk Configuration Parameters
+|===
+| Key | Description | Required | Default value
+| Host | Specifies the IP address or hostname of the target Splunk service. | | 127.0.0.1
+| Port | Specifies the TCP port of the target Splunk service. | | 8088
+| Splunk_Token | Specifies the Authentication Token for the HTTP Event Collector interface. | |
+| Splunk_Send_Raw | When enabled, specifies the record keys and values to be set in the top level of the map instead of under the `event` key. | | Off
+| HTTP_User | Specifies an optional username for Basic Authentication on HEC. | |
+| HTTP_Passwd | Specifies a password for the user defined in `HTTP_User`. | |
+|===
+
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name        splunk
+    Match       *
+    Host        127.0.0.1
+    Port        8088
+    TLS         On
+    TLS.Verify  Off
+    Message_Key my_key
+```
+
+== Disable Log Forwarding for Specific Pods
+
+By default, Fluent Bit log forwarding is enabled on all pods (MuleApplication pods, Runtime Fabric pods and Infrastructure pods). 
+
+To disable log forwarding for specific pods, you can annotate it with `fluentbit.io/exclude`, as shown in the following example:
```

Comment:
To what does "it" refer: "log forwarding" or "specific pods"? If the latter, fix pronoun agreement.